File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    answer = [0, 0]
    
    while s != "1":
        length = len(s)
        count_0 = s.count('0')
        s = bin(length - count_0)[2:]
        answer[1] += count_0
        answer[0] += 1
    
    return answer

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.js
function solution(s) {
    const answer = [0, 0];
    
    while (s !== "1") {
        const len = s.length;
        const count_0 = [...s].filter(char => char === '0').length;
        s = (len - count_0).toString(2);
        answer[1] += count_0;
        answer[0] += 1;
    }
    
    return answer;
}
